packages/lib/src/vis/tiles/api.ts

```diff
@@ -0,0 +1,50 @@
+import type { NdArray } from 'ndarray';
+import type { Vector2 } from 'three';
+
+import type { TextureSafeTypedArray } from '../heatmap/models';
+import type { Size } from '../models';
+
+export function getLevelsSize(
+  size: Size,
+  tileSize: Size,
+  evenLevelSize = false
```

Comment:
This would better be called `roundToEven` since level size can be odd when `evenLevelSize = true`